# cd /data/openpilot && curl https://raw.githubusercontent.com/eFiniLan/openpilot-patches/master/0.5.12/disable_mapd.diff | git apply -v
#
# This patch disable map entirely so OP won't request data from OpenStreetMap to reduce mobile data usage.
#
===================================================================
--- a/selfdrive/manager.py
+++ b/selfdrive/manager.py
@@ -92,7 +92,7 @@
   "plannerd": "selfdrive.controls.plannerd",
   "radard": "selfdrive.controls.radard",
   "ubloxd": ("selfdrive/locationd", ["./ubloxd"]),
-  "mapd": "selfdrive.mapd.mapd",
+  # "mapd": "selfdrive.mapd.mapd",
   "loggerd": ("selfdrive/loggerd", ["./loggerd"]),
   "logmessaged": "selfdrive.logmessaged",
   "tombstoned": "selfdrive.tombstoned",
